### PR TITLE
prepare for release 4.9.4

### PR DESCRIPTION
**Pull request by raju-opti**

### CODE REVIEW

File: packages/optimizely-sdk/CHANGELOG.md [hunk 1]
```diff
@@ -5,9 +5,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
```
Comment (jaeopt) [inline packages/optimizely-sdk/CHANGELOG.md:8]: Is it intentional to remove this section? I understand this is to keep track of contents to be released.

File: packages/optimizely-sdk/CHANGELOG.md [hunk 2]
```diff
@@ -5,9 +5,10 @@ All notable changes to this project will be documented in this file.
...
+## [4.9.4]
```
Comment (jaeopt) [inline packages/optimizely-sdk/CHANGELOG.md:8]: Date is missing.

**Review by jaeopt — CHANGES_REQUESTED** [review]
A couple of small changes suggested. The change in CHANGELOG should be merged into master while other changes are already in master.

File: packages/optimizely-sdk/CHANGELOG.md (hunk 1 above)
Comment (raju-opti) [reply packages/optimizely-sdk/CHANGELOG.md:8]: I see. I'll keep this section then

File: packages/optimizely-sdk/CHANGELOG.md (hunk 2 above)
Comment (raju-opti) [reply packages/optimizely-sdk/CHANGELOG.md]: updated

File: packages/optimizely-sdk/CHANGELOG.md (hunk 1 above)
Comment (raju-opti) [reply packages/optimizely-sdk/CHANGELOG.md:8]: updated

**Pull request merged**